src/Nimiq/Transaction.cpp

```diff
@@ -42,6 +43,7 @@ std::vector<uint8_t> Transaction::getPreImage() const {
     encode32BE(vsh, data);
     data.push_back(NETWORK_ID);
     data.push_back(EMPTY_FLAGS);
+    data.push_back(0x00); // Sender Data size (+ 0 bytes of data)
```

Comment:
Could you please add a link to the source code somewhere at the beginning of the function?
https://github.com/nimiq/core-rs-albatross/blame/b8ed402c9096ffb54afea52347b91ab7831e75de/primitives/transaction/src/lib.rs#L582